### ISSUE TITLE
Crash Handler is disabled for binaries compiled with MinGW (includes official Windows binaries)

### ISSUE THREAD

**Issue by jamie-pate** [godotengine/godot#42540]
**Godot version:**

3.2.3-stable

**OS/device including version:**

Windows

**Issue description:**

When the exported game crashes, there's no stack trace or anything, even the debug build. It looks like the official templates are built from MinGW~32~ and therefore have this feature disabled.

It would be nice if the default export template had a way to print out or log crash info. It's not feasible to attach a debugger in the field, and creating a custom build for distribution is painful.

Either find a way to build these templates with MSVC, or find a way to print out the stack trace in MinGW?

```C++
// Crash handler exception only enabled with MSVC
#if defined(DEBUG_ENABLED) && defined(MSVC)
#define CRASH_HANDLER_EXCEPTION 1
```

**Steps to reproduce:**
Crash the game after exporting for windows with debug enabled.

**Minimal reproduction project:**

not sure how to deliberately crash :+1:

**Comment by Calinou** [comment]
> It looks like the official templates are built from MinGW32 and therefore have this feature disabled.

We use MinGW-w64 for official builds, not the ages old "MinGW32";

**Comment by jamie-pate** [comment]
"print out the stack trace in MinGW" would be good, I think the current behavior is silent failure. I'll be doing more exhaustive research on Monday.

It's been increasingly difficult to track down crashes etc on disparate hardware without the live, in person test sessions that we were able to run pre-covid :)

**Comment by fire** [comment]
See also https://github.com/godotengine/godot-proposals/issues/1342#issuecomment-702189368

**Comment by alvinhochun** [comment]
This should be closed by #61006, right?

**Comment by Calinou** [comment]
Indeed, this was implemented by https://github.com/godotengine/godot/pull/61006.
